.github/workflows/lint.yml

```diff
@@ -26,14 +14,28 @@ jobs:
       - uses: dprint/check@v2.2
         with:
           dprint-version: '0.34.1'
-  shfmt:
+  shellscript:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
       - uses: actions/checkout@v3
-        # https://github.com/asdf-vm/actions/blob/171cc3570019f3a69ba10ae269ab46990936fa9f/README.md?plain=1#L128-L165
-      - name: asdf_install
-        uses: asdf-vm/actions/install@v1
-        with:
-          tool_versions: 'shfmt 3.6.0'
-      - run: 'shopt -s globstar && shfmt --diff ./**/*.bash'
+      # Using nix is best, but it is slow even if installing tiny binary
+      - name: Install rtx
+        run: |
+          mkdir -p ~/.local/share/rtx/bin
+          curl -L https://github.com/jdxcode/rtx/releases/download/v1.12.1/rtx-v1.12.1-linux-x64 > ~/.local/share/rtx/bin/rtx
+          chmod +x ~/.local/share/rtx/bin/rtx
+          ~/.local/share/rtx/bin/rtx version
+          echo "$HOME/.local/share/rtx/bin" >> $GITHUB_PATH
```

Comment:
https://github.com/jdxcode/rtx/issues/116 👀

Reply:
lmk how well this works for you: https://github.com/marketplace/actions/rtx-install

Reply:
Applied in https://github.com/kachick/dotfiles/pull/153, thanks for the great project! 🎉